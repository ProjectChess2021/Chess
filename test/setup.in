setup
+ r a5
+ n g6
+ K d2
done
+ k c4
+ k b4
done
- k b4
+ P b3
done
- P b3
+ P b8
done
- P b8
+ P b1
done
- P b1
= white
done
