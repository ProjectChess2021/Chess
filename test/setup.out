8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _ _
5 r _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ K _ _
1 _ _ _ _ 

  abcdefgh
No black king is detected. Please try again.
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _k_ _ _
3 _ _ _ _ 
2  _ K _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  kk_ _ _
3 _ _ _ _ 
2  _ K _ _
1 _ _ _ _ 

  abcdefgh
Cannot have multiple black king. Please try again.
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _k_ _ _
3 _ _ _ _ 
2  _ K _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _k_ _ _
3 _P_ _ _ 
2  _ K _ _
1 _ _ _ _ 

  abcdefgh
Cannot put the black king in check. Please try again.
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _k_ _ _
3 _ _ _ _ 
2  _ K _ _
1 _ _ _ _ 

  abcdefgh
8  P _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _k_ _ _
3 _ _ _ _ 
2  _ K _ _
1 _ _ _ _ 

  abcdefgh
Cannot place the pawn on the first or last row of the board. Please try again.
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _k_ _ _
3 _ _ _ _ 
2  _ K _ _
1 _ _ _ _ 

  abcdefgh
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _k_ _ _
3 _ _ _ _ 
2  _ K _ _
1 _P_ _ _ 

  abcdefgh
Cannot place the pawn on the first or last row of the board. Please try again.
8  _ _ _ _      MoveHistory
7 _ _ _ _ 
6  _ _ _n_
5 r _ _ _ 
4  _k_ _ _
3 _ _ _ _ 
2  _ K _ _
1 _ _ _ _ 

  abcdefgh

Setup successful and it's white's turn to go next.
